prebid-server/endpoints/openrtb2/video.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: page_v2
+sidebarType: 5
+title: Prebid Server | Endpoints | Video
+
+---
+
+# Prebid Server Video Endpoint
+
+This document describes the behavior of the Prebid Server `video` endpoint in detail.
+
+## Video Endpoint
+ `POST /openrtb2/video`
+
+## Overview
+
+Prebid Server (PBS) supports competitve separation for ad pod display from primary ad servers such as Freewheel and DFP.  PBS, on receiving this endpoint, will pass default settings and `stored_request` parameters to one or more OpenRTB request, which will then be sent to the appropriate SSPs. 
+
+The process is similar to [PrebidJS client side](/prebid-video/video-long-form.html) support for this feature.
+
+### Process
+
+1. Application makes request for a video stream. 
+2. An SSAI Server sends an OpenRTB request to PBS.
+3. PBS sends a request for bids to selected demand partners by sending the OpenRTB request to them. 
+4. Demand partners return a bid response to PBS. If competitive seperation is enabled, PBS peforms [category translation](/dev-docs/modules/categoryTranslation.html) on each bid. Whether category translation is required or not, the bids are stored in prebid cache. 
+5. PBS generates key-value pairs that are comprised of price, industry and duration values, `hb_pb_industry_duration = 1200_399_30s`. These key-values are returned to the SSAI server as part of the OpenRTB response. 
+6. The SSAI server parses the returned key-values, appending them as a query string to the ad server request URL and submits the request. 
+7. The ad server returns the optimized pod. 
+8. The SSAI server requests the creatives from prebid cache. Once the creatives are received it stitches the content and ads together. 
+9. The stitched stream is returned to the application. 
```

Comment:
This is step 10 in the diagram, you skipped step 9 here.

Reply:
Fixed